JS/Planes/PassengerPlane.js

```diff
@@ -2,21 +2,18 @@ const Plane = require('./Plane');
 
 class PassengerPlane extends Plane {
 
-    //=================CONSTRUCTORS=================
-
     constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity, passengersCapacity) {
```

Comment:
This constructor has too many arguments. It would be a good idea to pass a single object to a constructor. This object will have properties: model, maxSpeed, etc.

Reply:
Done